libs/cli/helper.go

```diff
@@ -1,29 +1,16 @@
 package cli
 
 import (
-	"bytes"
-	"fmt"
-	"io"
 	"os"
-	"path/filepath"
 
 	"github.com/spf13/cobra"
 )
 
-// WriteConfigVals writes a toml file with the given values.
-// It returns an error if writing was impossible.
-func WriteConfigVals(dir string, vals map[string]string) error {
-	data := ""
-	for k, v := range vals {
-		data += fmt.Sprintf("%s = \"%s\"\n", k, v)
-	}
-	cfile := filepath.Join(dir, "config.toml")
-	return os.WriteFile(cfile, []byte(data), 0600)
-}
-
 // RunWithArgs executes the given command with the specified command line args
 // and environmental variables set. It returns any error returned from cmd.Execute()
-func RunWithArgs(cmd Executable, args []string, env map[string]string) error {
+//
+// This is only used in testing.
+func RunWithArgs(cmd *cobra.Command, args []string, env map[string]string) error {
```

Comment:
Worthwhile maybe to rename it to reflect that intent?  Spitballing: `RunWithArgsForTesting`? (Yes, that name is obnoxious, intentionally so 🙂, but YMMV)

Reply:
I'm inclined to leave it as is for the moment, just because I think that we'll be able to delete it entirely in short order, and it's just a touch gnarly in its current use at the moment.)